katello/katello.spec

```diff
@@ -85,6 +85,13 @@ install -m 755 %{SOURCE3} %{buildroot}%{_sysconfdir}/cron.weekly/katello-remove-
 install -m 755 %{SOURCE10} %{buildroot}%{_sysconfdir}/cron.weekly/katello-clean-empty-puppet-environments
 install -m 755 %{SOURCE12} %{buildroot}%{_sysconfdir}/cron.daily/katello-repository-publish-check
 
+# symlink script libraries
+mkdir -p %{_datarootdir}/katello
+ln -s lib/restore.rb %{_datarootdir}/katello/restore.rb
+ln -s lib/backup.rb %{_datarootdir}/katello/backup.rb
+ln -s lib/hostname-change.rb %{_datarootdir}/katello/hostname-change.rb
+ln -s lib/script-helper.rb %{_datarootdir}/katello/script-helper.rb
```

Comment:
All of these new files will need to be included as Source at the top of the spec.

Reply:
Instead of symlinks, you can see down below, starting at line 98, you can just have them installed to the location you expect them at.

Reply:
Take a look at line 135 -- you'll need to include all of these so they get laid down on disk.